src/CMakeLists.txt

```diff
@@ -204,7 +204,7 @@ set(
   session_concurrency/response_json_utils.h
   session_concurrency/session_concurrency_manager.h
   rapidjson_utils.h
-  session_concurrency_mode.h
+  inference_load_mode.h
```

Comment:
Instead of each inference load mode having its own bool stating whether it is enabled or not, I just made one enum to hold which of the inference load modes was chosen via its CLI option.